### PR TITLE
[release-4.13] [manual] OCPBUGS-17794: rps: use default rps mask kernel API

### PR DESCRIPTION
**Pull request by Tal-or**
Manual cherry pick of: https://github.com/openshift/cluster-node-tuning-operator/pull/650 
Plus the following fixes for the e2e test: https://github.com/openshift/cluster-node-tuning-operator/pull/691, https://github.com/openshift/cluster-node-tuning-operator/pull/703

### CODE REVIEW

File: assets/performanceprofile/configs/99-default-rps-mask.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# We should apply the rule on the virtual interfaces of the host, because of the RPS mask that would be consulted,
-# is the one on the RX side of the veth in the host.
+# Apply the RPS mask on the virtual interfaces of the host by default, becasue
```
Comment (yanirq) [inline assets/performanceprofile/configs/99-default-rps-mask.conf:1]: @Tal-or  please note that there was a rename here, is that correct ?

Comment (Tal-or) [reply assets/performanceprofile/configs/99-default-rps-mask.conf:1]: yes why not?

**Pull request merged**